recurring_payments_stripe/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Recurring Payments with Stripe",
+    "version": "16.0.1.0.0",
+    "summary": """ Recurring Payments with Stripe """,
```

Comment:
Not sure about the spacing before and after the `"""`.

Reply:
Fixed spaces before and after within summary content